libflux/src/flux/semantic/builtins.rs

```diff
@@ -301,6 +301,178 @@ pub fn builtins() -> Builtins<'static> {
+                        columns: [string]
+                    ) -> [t1]
+                "#),
+                "cumulativeSum" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?columns: [string]
+                    ) -> [t1]
+                "#),
+                "derivative" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?unit: duration,
+                        ?nonNegative: bool,
+                        ?columns: [string],
+                        ?timeColumn: string
+                    ) -> [t1]
+                "#),
+                "difference" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?nonNegative: bool,
+                        ?columns: [string],
+                        ?keepFirst: bool
+                    ) -> [t1]
+                "#),
+                "distinct" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?column: string
+                    ) -> [t1]
+                "#),
+                "drop" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t1: Row, t2: Row (
+                        <-tables: [t0],
+                        ?fn: (column: string) -> bool,
+                        ?columns: t1
+                    ) -> [t2]
+                "#),
+                "duplicate" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        column: string,
+                        as: string
+                    ) -> [t1]
+                "#),
+                "duration" => Node::Builtin("forall [t0] (v: t0) -> duration"),
+                "elapsed" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        ?unit: duration,
+                        ?timeColumn: string,
+                        ?columnName: string
+                    ) -> [t1]
+                "#),
+                "exponentialMovingAverage" => Node::Builtin(r#"
+                    forall [t0, t1] where t0: Row, t1: Row (
+                        <-tables: [t0],
+                        n: int
+                    ) -> [t1]
+                "#),
+                "false" => Node::Builtin("forall [] () -> bool"),
+                "fill" => Node::Builtin(r#"
+                    forall [t0, t1, t2] where t0: Row, t2: Row (
+                        <-tables: [t0],
+                        ?column: string,
+                        value: [t1],
+                        usePrevious: bool
+                    ) -> [t2]
+                "#),
```

Comment:
This can almost be assigned a valid type, if only it didn't take the `?column` parameter. What you have here is correct, it's just another example of column arguments invalidating the type.